python/orca/src/bigdl/orca/learn/pytorch/callbacks.py

```diff
@@ -0,0 +1,101 @@
+from abc import abstractmethod
+
+
+class Callback(object):
+    def __init__(self):
+        self.model = None
+        self.params = None
+
+    @abstractmethod
+    def on_batch_begin(self, batch, logs=None):
+        """
+        A backwards compatibility alias for `on_train_batch_begin`.
```

Comment:
which one do you expect user to override? I think only `on_batch_begin` and `on_batch_end` are enough and we can remove `on_train_batch_beigin` and `on_train_batch_end`